src/inmanta/agent/handler.py

```diff
@@ -354,7 +394,10 @@ def update_changes(  # noqa: F811
     def changes(self) -> Dict[str, AttributeStateChange]:
         return self._changes
 
-    def log_msg(self, level: int, msg: str, args: Sequence[object], kwargs: Dict[str, object]) -> None:
+    def log_msg(self, level: int, msg: str, args: abc.Sequence[object], kwargs: abc.Mapping[str, object]) -> None:
+        LOGGER.warning(
```

Comment:
Are you planning to drop this from master after merging? Otherwise I'd add a comment as to why it's here and when it can be dropped.

Reply:
Yes :+1: